### ISSUE TITLE
center loss updating

### ISSUE THREAD

**Issue by hbellafkir** [davidsandberg/facenet#1114]
hey,

By using the tf.glabal_variables() as var_list to compute and apply the gradients here:
`train_op = facenet.train(total_loss, global_step, args.optimizer, 
            learning_rate, args.moving_average_decay, tf.global_variables())`
you ignoring the fact, that the center variable shouldn't be trainable. so the code is applying the gradients twice for the center loss. one time manually in facenet.center_loss() and one with the Optimizer. is that a bug?
